zjit/src/codegen.rs

```diff
@@ -880,6 +902,11 @@ fn gen_intern(asm: &mut Assembler, val: Opnd, state: &FrameState) -> Opnd {
     asm_ccall!(asm, rb_str_intern, val)
 }
 
+fn gen_string_bytesize(asm: &mut Assembler, recv: Opnd, state: &FrameState) -> Opnd {
+    gen_prepare_leaf_call_with_gc(asm, state);
+    asm_ccall!(asm, rb_zjit_string_bytesize, recv)
```

Comment:
Can we inline instructions that load the length? You can do the same thing as YJIT's `jit_rb_str_bytesize` here.